### PR TITLE
feat: Added variable trusted_role_actions to sub modules as a "Action of STS"

### LINKED ISSUE #392: Feat: Adding variable trusted_role_actions to other sub-modules
## Is your request related to a problem? Please describe.
Currently variable trusted_role_actions is only available at sub-module iam-assumable-role.

## Describe the solution you'd like.
 I'd like to have the same variable, in sub modules. For saml sub module default variable will be different.
- iam-assumable-roles
- iam-assumable-role-with-saml
- iam-assumable-roles-with-saml

## Describe alternatives you've considered.
None.

## Additional context
It's useful to use two additional action with STS action:
- sts:SetSourceIdentity
- sts:TagSession

But when you want to use them, you need to add them to all roles in AWS Account.

### PR DESCRIPTION
**Pull request by tcharewicz**
## Description
Orginaly I create issue with "Adding variable trusted_role_actions to other sub-modules"
    - iam-assumable-roles
    - iam-assumable-role-with-saml
    - iam-assumable-roles-with-saml

- Resolves #392

At the time of PR creation I found that, in sub module iam-assumable-role-with-saml this variable is used, but in different way comparing to sub module iam-assumable-role. 

In module with iam-assumable-role-with-saml is used as a "Extra Actions of STS"
```
variable "trusted_role_actions" {
  description = "Extra Actions of STS"
  type        = list(string)
  default     = [""]
}
``` 

In module with iam-assumable-role is used as a "Actions of STS"
```
variable "trusted_role_actions" {
  description = "Actions of STS"
  type        = list(string)
  default     = ["sts:AssumeRole"]
}
```

That way finally PR is adding this variable as a "Extra Actions of STS" to sub modules:
- iam-assumable-role
- iam-assumable-roles
- iam-assumable-roles-with-saml

## Motivation and Context
It's useful to use two additional action with STS action:

    sts:SetSourceIdentity
    sts:TagSession

But when you want to use them, you need to add them to all roles in AWS Account.

## Breaking Changes
None, all orginal default action are preserved.

## How Has This Been Tested?
- [yes] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Review by bryantbiggs — CHANGES_REQUESTED** [review]

File: modules/iam-assumable-roles/main.tf [hunk 1]
```diff
@@ -75,7 +75,7 @@ data "aws_iam_policy_document" "assume_role" {
 
   statement {
     effect  = "Allow"
-    actions = ["sts:AssumeRole"]
```
[the remark this answers is not in the document]
Comment (tcharewicz) [reply modules/iam-assumable-roles/main.tf:78]: It's changed like you said. In this modules this variable is right now a list of STS actions. Please review PR.

**Review by bryantbiggs — APPROVED** [review]

File: modules/iam-assumable-roles/main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryantbiggs) [reply modules/iam-assumable-roles/main.tf:78]: apologies - it looks like we have/had multiple different methods in use across the modules here. reverted back to the concatting you had originally proposed

**Comment by antonbabenko** [comment]
This PR is included in version 5.23.0 :tada:

**Comment by tcharewicz** [comment]
@bryantbiggs Thx.

**Pull request merged**